client/a8c-for-agencies/components/items-dashboard/item-preview-pane/types.ts

```diff
@@ -41,6 +41,7 @@ export interface PreviewPaneProps {
 	addTourDetails?: { id: string; tourId: string };
 	itemPreviewPaneHeaderExtraProps?: ItemPreviewPaneHeaderExtraProps;
 	hideNavIfSingleTab?: boolean;
+	enforceTabsView?: boolean;
```

Comment:
Half-kidding: maybe it's time to recreate the components instead of patching A4A code like this 🤔

Reply:
Yes, and I'd prefer to use the [TabList](https://wordpress.github.io/gutenberg/?path=/story/components-tabs--size-and-overflow-playground) component from Gutenberg in the future